.github/workflows/lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Run linter
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.63.4
+          version: v1.64.5
```

Comment:
Could we open a PR for upgrade the lint and another for the Go
We also need check all places for the linter

Reply:
Thanks @camilamacedo86 
I got it
Will change it in next PR

Reply:
Thank you a lot for check this one !!!!